### DISCUSSION TITLE
On This Day type search or filter?

### DISCUSSION THREAD

**Discussion by bencollinz** [bpatrik/pigallery2#648] [Q&A]
Is there a way to do an On This Day type of setup? With a search or filter or something else I'm not finding?
I realize I could choose today's date and then add the years going back to whatever I want but is there an easier way to do it without manual intervention?
That's really the only thing I miss using this.

Thank you!

**Comment by bpatrik** [comment]
Hi,
There is no such feature at the moment.

**Reply by bpatrik** [reply]
This is available with the latest nightly

**Reply by bencollinz** [reply]
I will spin it back up and try. Thank you.

**Reply by bencollinz** [reply]
Works great! Thank you. Is there a way to autoplay an album from a URL? 
`http://192.168.0.200:3090/search/%7B%22type%22:60,%22daysLength%22:1,%22frequency%22:3%7D`

**Comment by bpatrik** [comment]
That is not supported at the moment.

-- Sorry for being brief, sent from my phone.
